### DISCUSSION TITLE
cloudshell

### DISCUSSION THREAD

**Discussion by manicdictator** [sherlock-project/sherlock#1344] [Q&A]
not to sound like an idiot... how exactly do I download this onto cloudshell - I am not sure from the destructions and am new to this site :)

**Comment by 546pvp** [comment]
Just use “git clone [url]
Then open the directory with “cd”
And run “python3 Sherlock [username]”
I hope this helps. If it not works reply to this answer.

**Comment by Cfomodz** [comment]
```bash
pip install sherlock-project --user
export PATH="$HOME/.local/bin:$PATH"
```
